iOS/Swift/README.md

```diff
@@ -55,59 +52,58 @@ class app_widgetContainer {
 }
 ```
 
-For functions and init methods, prefer named parameters for all arguments unless the context is very clear. Include external parameter names if it makes function calls more readable.
+For functions and init methods, prefer *named parameters* for all arguments unless the context is very clear. Include external parameter names if it makes function calls more readable.
 
 ```swift
-func dateFromString(dateString: String) -> NSDate
-func convertPointAt(column column: Int, row: Int) -> CGPoint
-func timedAction(afterDelay delay: NSTimeInterval, perform action: SKAction) -> SKAction!
+func date(from string: String) -> Date
+func convertPoint(at column: Int, in row: Int) -> CGPoint
+func timedAction(after delay: NSTimeInterval, perform action: SKAction) -> SKAction!
 
 // would be called like this:
-dateFromString("2014-03-14")
-convertPointAt(column: 42, row: 13)
-timedAction(afterDelay: 1.0, perform: someOtherAction)
-```
+date(from: "2014-03-14")
+convertPoint(at: 42, in: 13)
+timedAction(after: 1.0, perform: someOtherAction)
+// Improvement on this would be to also name the variables
+let dateString = "2014-03-14"
+let column = 42
+let row = 13
+let time = 1.0
+date(from: dateString)
+convertPoint(at: column, in: row)
+timedAction(after: time, perform: someOtherAction)
+```
+The code above could mentions how to avoid spreading Ints, Strings, … around. Prefer parameters as variables.
```

Comment:
n code you use a lot of `10.0` or `"string"`. This is handy to get things going. 
But after a while this becomes dangerous. Also the naming conventions of `Swift 3.0` prefer
shorter but descriptive naming. If you use variable name over `primitive types` the code
becomes more descriptive. As examplified above.